app/views/contact_details.scala.html

```diff
@@ -57,13 +58,17 @@
                     labelClasses = "govuk-label",
                 )
 
-                @exportsInputText(
-                    field = form("phoneNumber"),
-                    labelKey = "contactDetails.phoneNumber",
-                    labelClasses = "govuk-label",
-                    inputmode = "numeric",
-                    pattern = "[0-9]*",
-                )
+                @govukInput(Input(
+                    id = form("phoneNumber").id,
+                    name = form("phoneNumber").name,
+                    value = form("phoneNumber").value,
+                    label = Label(content = Text(messages("contactDetails.phoneNumber")), classes = "govuk-label"),
+                    errorMessage = form("phoneNumber").error.map(err => ErrorMessage(content = Text(messages(err.message)))),
+                    classes = "govuk-!-width-two-thirds",
+                    pattern = Some("[0-9]*"),
+                    inputmode = Some("numeric"),
+                    inputType = "text"
+                ))
```

Comment:
Why not just add this `inputmode` as an optional parameter onto our `exportsInputText` component? :thinking: